src/content/admin/resource-manager.mdx

```diff
@@ -33,36 +33,20 @@ It calculates the average usage and automatically adjusts the resource requests
 1. Automatic Application: When a new pod is created, the recommended resource requests are automatically applied unless overridden the annotation `dev.okteto.com/resource-manager: false`
 
 ## Enabling and Configuring the Resource Manager
-<!-- ### Enabling the Resource Manager
-The Resource Manager is controlled through the Helm chart configuration. By default, it is enabled with the following settings:
-
-``` yaml
-resourceManager:
-  enabled: true
-  schedule: 300
-  deletePeriod: 15
-  priorityClassName:
-    labels:
-	annotations:
-	resources:
-	recommendations:
-		weight:
-		correction:
-		min:
-			cpu:
-			memory:
-``` -->
+### Enabling the Resource Manager
+You can enable the Resource Manager through the Helm chart configuration. Refer to our [Helm configuration documentation](self-hosted/helm-configuration.mdx#resourcemanager) for available options.
```

Comment:
The RM is enabled by default

Reply:
Is it already enabled for all instances upgrading to 1.26?

Reply:
yes, it's is installed by default and in "manual" mode